README.md

```diff
@@ -70,8 +70,8 @@ The emsdk toolchain is used to cross-compile Vireo to Vireo.js to run in Node.js
 4. From either the extracted folder or the cloned emsdk repository run the following commands:
 
    ```console
-   emsdk install sdk-1.38.30-64bit
-   emsdk activate sdk-1.38.30-64bit
+   emsdk install 1.39.5-fastcomp
```

Comment:
As emscripten seems to be going towards  LLVM WebAssembly backend (ones with "-upstream" suffix),  should we try to switch to that instead of old "fastcomp"? I am not sure whether LLVM WebAssembly backend offers asm-js (it may not, I guess) and not sure whether we need to be able to build (as needed basis from now on) !? Thoughts?

Reply:
We should definitely consider it as a separate change. The goal of this change was to switch us to releases from the [new build infrastructure](https://groups.google.com/forum/#!msg/emscripten-discuss/WhDtqVyW_Ak/8DfDnfk0BgAJ) and be on the latest fastcomp backend so that if for some reason we do file issues to emscripten, etc it will be easier to merge them in.

The fastcomp backend is supported in maintenance mode, but we should migrate to the llvm backend when possible